src/contracts/misc/GhoAaveSteward.sol

```diff
@@ -0,0 +1,328 @@
+    uint256 variableRateSlope2
+  ) external onlyRiskCouncil notTimelocked(_ghoTimelocks.ghoBorrowRateLastUpdate) {
+    _validateRatesUpdate(
+      optimalUsageRatio,
+      baseVariableBorrowRate,
+      variableRateSlope1,
+      variableRateSlope2
+    );
+    _updateRates(optimalUsageRatio, baseVariableBorrowRate, variableRateSlope1, variableRateSlope2);
+
+    _ghoTimelocks.ghoBorrowRateLastUpdate = uint40(block.timestamp);
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function updateGhoBorrowCap(
+    uint256 newBorrowCap
+  ) external onlyRiskCouncil notTimelocked(_ghoTimelocks.ghoBorrowCapLastUpdate) {
+    DataTypes.ReserveConfigurationMap memory configuration = IPool(
+      IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPool()
+    ).getConfiguration(GHO_TOKEN);
+    uint256 currentBorrowCap = configuration.getBorrowCap();
+    require(
+      _isDifferenceLowerThanMax(currentBorrowCap, newBorrowCap, currentBorrowCap),
+      'INVALID_BORROW_CAP_UPDATE'
+    );
+
+    _ghoTimelocks.ghoBorrowCapLastUpdate = uint40(block.timestamp);
+
+    IPoolConfigurator(IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPoolConfigurator())
+      .setBorrowCap(GHO_TOKEN, newBorrowCap);
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function updateGhoSupplyCap(
+    uint256 newSupplyCap
+  ) external onlyRiskCouncil notTimelocked(_ghoTimelocks.ghoSupplyCapLastUpdate) {
+    DataTypes.ReserveConfigurationMap memory configuration = IPool(
+      IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPool()
+    ).getConfiguration(GHO_TOKEN);
+    uint256 currentSupplyCap = configuration.getSupplyCap();
+    require(
+      _isDifferenceLowerThanMax(currentSupplyCap, newSupplyCap, currentSupplyCap),
+      'INVALID_SUPPLY_CAP_UPDATE'
+    );
+
+    _ghoTimelocks.ghoSupplyCapLastUpdate = uint40(block.timestamp);
+
+    IPoolConfigurator(IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER).getPoolConfigurator())
+      .setSupplyCap(GHO_TOKEN, newSupplyCap);
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function setRiskConfig(
+    Config calldata riskConfig
+  ) external onlyRiskCouncil notTimelocked(_ghoTimelocks.riskConfigLastUpdate) {
+    _riskConfig = riskConfig;
+    _ghoTimelocks.riskConfigLastUpdate = uint40(block.timestamp);
+    emit RiskConfigSet(riskConfig);
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function getRiskConfig() external view returns (Config memory) {
+    return _riskConfig;
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function getGhoTimelocks() external view returns (GhoDebounce memory) {
+    return _ghoTimelocks;
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function RISK_COUNCIL() public view override returns (address) {
+    return COUNCIL;
+  }
+
+  function _updateRates(
+    uint256 optimalUsageRatio,
+    uint256 baseVariableBorrowRate,
+    uint256 variableRateSlope1,
+    uint256 variableRateSlope2
+  ) internal {
+    DefaultReserveInterestRateStrategyV2 newRateStrategy = new DefaultReserveInterestRateStrategyV2(
```

Comment:
why deploying a new one? we can use the current strategy afaik

Reply:
I saw the newer risk steward deploying a new strategy. Do we want the option to deploy a new one for the flexibility?

Reply:
i think using the current one is fine

Reply:
Done